aiodns/__init__.py

```diff
@@ -62,12 +72,9 @@ def __init__(self, nameservers: Optional[Sequence[str]] = None,
                 except ModuleNotFoundError:
                     raise RuntimeError(
                         'aiodns needs a SelectorEventLoop on Windows. See more: https://github.com/saghul/aiodns/issues/86')
```

Comment:
Instead of that PR, should we maybe direct folks rather to the README.md which has the additional information about the thread safe version? :thinking:

Reply:
adjusted in 4f60a66a6cc8d5e090840b7632ade6ee7f17afc9